<?php 
$ex = $this->exception;
/**
 * @var ex Zend_Exception
 */
?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"; "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd> 
<html xmlns="http://www.w3.org/1999/xhtml"> 
<head>  
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" /> 
  <title>Zend Framework Default Application</title> 
</head> 
<body> 
  <h1>Ups - Der skete en fejl...</h1> 
  <h2><?= $this->message ?></h2> 

  <?/* if ('development' == APPLICATION_ENV or true):*/ ?>
 
  <h3>Exception information:</h3> 
  <p> 
      <b>Message:</b> <?= nl2br($this->exception->getMessage()) ?> 
  </p> 
  <p>
  	<b>Type:</b><?
/**
 * @var ex Zend_Exception
 */
  	//$ex = new Zend_Exception();
  	switch (get_class($ex)) {
  		case 'Zend_Db_Table_Exception':
  			$zex = new Zend_Db_Table_Exception();
  			Zend_Debug::dump($ex,'Ex',true);
  		;
  		break;
  		
  		default:
  			echo 'Exeption type was <b>'.get_class($ex).'</b><br/>';
  			;
  		break;
  	}
  	echo nl2br($ex->__toString()) ?>
  </p>

   <h3>Stack trace:</h3> 
  <pre><?= $this->exception->getTraceAsString() ?> 
  </pre>   

  <h3>Request Parameters:</h3> 
  <pre><? var_dump($this->request->getParams()); ?> 
  </pre> 

   
    
</body> 
</html>
<?php 
phpinfo(); ?>